src/main/java/codeu/model/data/MentionList.java

```diff
@@ -0,0 +1,99 @@
+package codeu.model.data;
+
+import codeu.model.store.basic.UserStore;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class MentionList {
+    private Message message;
+    private  UserStore userStore;
+
+    public MentionList(Message message, UserStore userStore){
+        message = this.message;
+        userStore = this.userStore;
+    }
+
+    public List<Mention> getList(){
```

Comment:
Nice!  Naming, though: getMentions would be more clear because it tells you the purpose of the class.